src/views/masq_activating_modal.dot

```diff
@@ -0,0 +1,40 @@
+<div class="modal__masq_activating">
+  <div class="modal__masq_activating__header">
+    <div class="icon-masq-by-qwant modal__masq_activating__logo"></div>
+    <i class="icon-x modal__masq_activating__close" {{= click(this.close, this) }}></i>
+  </div>
+
+  <div class="modal__masq_activating__body">
+    {{? this.activating}}
+      <div class="modal__masq_activating__animation">
+        <video autoplay loop src="{{= window.location.origin }}/statics/images/masq/gifs/masq_loading.mp4"></video>
+      </div>
+      <h2 class="modal__masq_activating__message">{{= _('Patience, Masq s\'active...') }}</h2>
```

Comment:
this translation key needs to be defined in english :)

Reply:
:+1: